css/selectors/-webkit-resizer.json

```diff
@@ -0,0 +1,70 @@
+{
+  "css": {
+    "selectors": {
+      "-webkit-resizer": {
+        "__compat": {
+          "description": "::-webkit-resizer",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::-webkit-resizer",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": false,
+              "notes": "<a href='https://developer.microsoft.com/microsoft-edge/platform/status/webkitscrollbars/'>Under consideration</a>"
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "See <a href='https://bugzil.la/1432935'>bug 1432935</a>."
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "See <a href='https://bugzil.la/1432935'>bug 1432935</a>."
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "14"
+            },
+            "qq_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "samsunginternet_android": {
+              "version_added": true
+            },
+            "uc_android": {
+              "version_added": true
+            },
+            "uc_chinese_android": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "deprecated": false,
+            "experimental": false,
+            "standard_track": false
```

Comment:
Can these status blocks be sorted in the usual experimental-standard-deprecated order? This applies to each of the files in this PR.

Reply:
Done.

Also, it turns out that a lot of compat data uses the alphabetical order for&nbsp;the&nbsp;**status** block.